ui/src/config/section/infra.js

```diff
@@ -60,11 +62,12 @@ export default {
       component: () => import('@/views/infra/CpuSockets.vue')
     },
     {
-      name: 'managementserver',
-      title: 'label.management.servers',
-      icon: 'rocket',
-      permission: ['listManagementServers'],
-      columns: ['name', 'state', 'version']
+      name: 'metric',
+      title: 'label.metrics',
+      icon: 'bar-chart',
+      docHelp: 'adminguide/management.html#metrics',
+      permission: ['listDbMetrics', 'listUsageServerMetrics'],
+      component: () => import('@/components/view/Metrics.vue')
```

Comment:
should be moved to `./views/infra/Metrics.vue` ths component is not generic enough to warrant this location.